packages/react/src/Footer/MenuLink.client.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from "classnames";
+import { FC } from "react";
+
+export type Link = {
+  link: string;
+  text: string;
+  openInCurrentTab?: boolean;
+};
+
+type MenuLinkProps = {
+  links: Link[];
+  isAboutOpen: boolean;
```

Comment:
peut être rendre la prop `isAboutOpen `facultative et de la définir à `false `par défaut
```suggestion
  isAboutOpen?: boolean;
```

PS: je ne suis pas fan du namming mais j'ai pas mieux pour le moment